Core/src/org/sleuthkit/autopsy/directorytree/ExtractUnallocAction.java

```diff
@@ -69,14 +69,14 @@
     private long currentImage = 0L;
     private final boolean isImage;
 
-    public ExtractUnallocAction(String title, Volume volume) {
+    public ExtractUnallocAction(String title, Volume volume) throws NoCurrentCaseException {
```

Comment:
Thinking about this some more, rather than propagate the NoCurrentCaseException from the constructors, lets catch and log it and cal setEnabled(false) on the action so it would be grayed out, but not omitted from the node context menu.